roles/openshift_excluder/tasks/status.yml

```diff
@@ -20,13 +20,15 @@
 - name: Update to latest excluder packages
   package:
     name: "{{ openshift.common.service_type }}-excluder"
+    state: latest
```

Comment:
For atomic-openshift-excluder/origin-excluder this doesn't matter as those are unversioned excludes.
For atomic-openshift-docker-excluder it's ok because they're bound by the version of the excluder that's specific to a given release, we should never ship a new excluder to the same repo which would introduce a docker version that's not compatible.
For origin-docker-excluder this will be problematic right now because multiple versions are in one repo however that won't be the case for 3.6 (I think?). However, we've only provided excluders for 1.4 and soon 1.5 which both support docker 1.12.

So I think this is OK.